.azurepipelines/publish-maven-1ES.yml

```diff
@@ -74,14 +74,43 @@ extends:
           inputs:
             tasks: publish
             publishJUnitResults: false
-        - task: SFP.release-tasks.custom-build-release-task.EsrpRelease@4
-          displayName: ESRP Release
+      
+      - job: extractAdditonalData 
+        dependsOn: [Phase_1]

```

Comment:
It seems to me that this job does not depend on the build job

Reply:
It does since i need to use the artifacts produced from the first job and when it finishes i use them in the secon job so that's why dependsOn is needed.